ebms-payload/src/main/kotlin/no/nav/emottak/payload/crypto/PayloadSignering.kt

```diff
@@ -34,7 +37,7 @@ fun payloadSigneringConfig() =
             )
         else ->
             FileKeyStoreConfig(
-                keyStoreFilePath = getEnvVar("KEYSTORE_FILE_SIGN", "xml/signering_keystore.p12"),
```

Comment:
Er default en keystore vi bruker i Q1 for testing? Eller er tanken at man bruker dette lokalt eller i testene man kjører?

Reply:
"else->"
refererer til ikke prod/dev miljø, så implisitt lokalt.
Dersom du ikke har en KEYSTORE_FILE_SIGN så refererer default verdien til fila som ligger i test/resources/keystore.